toolkit/offchain/src/csl.rs

```diff
@@ -619,6 +637,39 @@ impl TransactionBuilderExt for TransactionBuilder {
 	}
 }
 
+pub(crate) enum Script {
```

Comment:
Maybe it deserves own `script` module? It wraps pure CSL in one variant only and maybe we'll wrap it like we wrap PlutusScript in the future. Just let me know if you want it here or in a separate mod.